code/math/staticrand.cpp

```diff
@@ -37,19 +32,20 @@ void init_semirand()
  */
 int static_rand(int num)
 {
-	int	a, b, c;
-
-	if (num < 0) 
+	if (num < 0)
 		num *= -1;
 
 	if (!Semirand_inited)
 		init_semirand();
 
-	a = num & (SEMIRAND_MAX - 1);
-	b = (num >> SEMIRAND_MAX_LOG) & (SEMIRAND_MAX - 1);
-	c = (num >> (2 * SEMIRAND_MAX_LOG)) & (SEMIRAND_MAX - 1);
+	const unsigned int num_unsigned = num;
+	auto a = num_unsigned & (SEMIRAND_MAX - 1);
+	auto b = (num_unsigned >> SEMIRAND_MAX_LOG) & (SEMIRAND_MAX - 1);
+	auto c = (num_unsigned >> (2 * SEMIRAND_MAX_LOG)) & (SEMIRAND_MAX - 1);
 
-	return Semirand[a] ^ Semirand[b] ^ Semirand[c];
+	return Semirand[a]
+			^ Semirand[b]
+			^ Semirand[c];
```

Comment:
These lines are now spread out for no reason.

Reply:
narf, I'm to used to having an auto-formatter on save.